src/App.tsx

```diff
@@ -29,6 +138,12 @@ function App() {
             Learn React
           </a>
           <MyTemplate name="NASA" paragraph1="agsgahuoaoa aung aa iabng ka " paragraph2="aggraathh shsh siklesa lsjh ls" image={nasaLogo} />
```

Comment:
Instead of adding the paragraphs as properties you could use component composition

https://felixgerschau.com/react-component-composition/